recipes/phabox/meta.yaml

```
@@ -34,7 +34,7 @@ requirements:
     - numpy >=1.26
     - pandas >=2
     - prodigal-gv >=2.11.0
-    - pyarrow >=16
+    - pyarrow =16.1
```

```suggestion
    - pyarrow 16.*
```

would that make sense?

---

I checked the pyarrow channel, seems there are only two version for 16: 16.1 and 16.0

I have just done a quick test for the 16.0 and it can also pass the test. I will update the yaml accordingly. Thanks.